postgres-appliance/scripts/configure_spilo.py

```diff
@@ -699,8 +699,9 @@ def write_wale_environment(placeholders, prefix, overwrite):
 
     wale = defaultdict(lambda: '')
     for name in ['WALE_ENV_DIR', 'SCOPE', 'WAL_BUCKET_SCOPE_PREFIX', 'WAL_BUCKET_SCOPE_SUFFIX',
-                 'WAL_S3_BUCKET', 'WAL_GCS_BUCKET', 'WAL_GS_BUCKET', 'WAL_SWIFT_BUCKET', 'BACKUP_NUM_TO_RETAIN'] +\
-            s3_names + swift_names + gs_names + walg_names:
+                 'WAL_S3_BUCKET', 'WAL_GCS_BUCKET', 'WAL_GS_BUCKET', 'WAL_SWIFT_BUCKET', 'BACKUP_NUM_TO_RETAIN',
+                 'WALG_AZ_PREFIX'] +\
+            s3_names + swift_names + gs_names + walg_names + azure_names:
```

Comment:
Lets merge these two lines, so it look nicely.
```suggestion
                 'WALG_AZ_PREFIX'] + s3_names + swift_names + gs_names + walg_names + azure_names:
```